metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-systemd",
-  "version": "6.3.1-rc0",
+  "version": "6.3.1",
```

Comment:
There are also some enhancements listed in the changelog, so it has to be a minor release. Please update it to 6.4.0 and regenerate the changelog.